src/utils/axiosUtils.ts

```diff
@@ -0,0 +1,28 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import HttpsProxyAgent from 'https-proxy-agent';
+
+/**
+ * This function is called to better handle axios.get calls with logic that allows
+ * the use of proxies. Not needed in tests unless specifically testing with proxies.
+ * Check https://github.com/axios/axios/issues/3459#issuecomment-766171276 for more info.
+ * @param url {string} - string representation of url to get
+ * @param responseType {any} - optional parameter to change the data type needed from get
+ * (ex. arraybuffer). In default it returns JSON
+ */
+export async function axiosGet(url: string, responseType?: any): Promise<AxiosResponse<any>> {
+  const httpsProxy = process.env.HTTPS_PROXY;
+  const axiosOptions: AxiosRequestConfig = {};
+  if (httpsProxy) {
+    // https://github.com/axios/axios/issues/3459
+    axiosOptions.httpsAgent = new (HttpsProxyAgent as any)(httpsProxy);
+    axiosOptions.proxy = false;
+  }
+  if (typeof responseType !== 'undefined') {
```

Comment:
I think it would be a little cleaner to just have this line be `if (responseType) {`. I think this might be a JavaScript convention, and if not, I think it's at least something we tend to do. Since `undefined` is falsy, this if statement will evaluate to true as long as `responseType` is defined.

Reply:
Made this change, thanks!